prisma/migrations/20231204145218_job_lock/migration.sql

```diff
@@ -0,0 +1,9 @@
+-- CreateTable
+CREATE TABLE "job_run" (
+    "job_name" TEXT NOT NULL,
+    "startedAt" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "endedAt" TIMESTAMP(3),
```

Comment:
Why is started a `TIMESTAMP(6)` and ended a `TIMESTAMP(3)`?

Reply:
Whoops, missed the annotation